### ISSUE TITLE
Period: months/years not counted in `total_seconds()`

### ISSUE THREAD

**Issue by zeebonk** [sdispater/pendulum#387]
Pendulum version: 2.0.5, possibly related to #346

**Context**
When working with classes/functions/APIs that expect some period be passed as number of seconds, I like to use a Duration + the `total_seconds()` function to clearly express what kind of period I'm passing.

**Problem**
I'm running into a situation where I want a period of a year in total seconds, but to my surprise I get 0 as a result. While experimenting I found that the same applies for months. Weeks and smaller seem to work as expected.

**Examples**
```py
In [9]: pendulum.Duration(years=1).total_seconds()
Out[9]: 0.0

In [10]: pendulum.Duration(months=1).total_seconds()
Out[10]: 0.0

In [11]: pendulum.Duration(days=1).total_seconds()
Out[11]: 86400.0
```

**Comment by Senpos** [comment]
#389 
No years/months are mentioned there, but looks similar

**Comment by zeebonk** [comment]
@Senpos that is not related to this issue, but I could answer the answer :)

**Comment by ab-w** [comment]
Any update? This bug can bite users if they're not careful.

**Comment by ALERTua** [comment]
Here's a quick way to reproduce this:
`
import pendulum
a = pendulum.duration(seconds=200000000)
a.in_words()
330 weeks 4 days 19 hours 33 minutes 20 seconds
`
The output should contain months and years.

**Comment by rnovacek** [comment]
I think this can be closed, it was fixed by PR #482.

**Comment by zeebonk** [comment]
Confirmed resolved with pendulum==3.0.0 and Python==3.11.4:

```
Python 3.11.4 (main, Sep 26 2023, 14:43:59) [Clang 15.0.0 (clang-1500.0.40.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> import pendulum
>>> pendulum.Duration(years=1).total_seconds()
31536000.0
>>> pendulum.Duration(months=1).total_seconds()
2592000.0
```
